src/schemas/table_schemas/designed_sequences.json

```diff
@@ -0,0 +1,136 @@
+                    "test",
+                    "variant positive control",
+                    "variant negative control",
+                    "element active control",
+                    "element inactive control"
+                ]
+            }
+        },
+        {
+            "name": "source",
+            "type": "string",
+            "description": "free-form description of the origin of the sequence"
+        },
+        {
+            "name": "ref",
+            "type": "string",
+            "description": "reference sequence, e.g. GRCh38"
+        },
+        {
+            "name": "chr",
+            "type": "string",
+            "description": "reference chromosome"
+        },
+        {
+            "name": "start",
+            "type": "integer",
+            "description": "0-based position of the left-most position of sequence with respect to the reference chromosome",
+            "constraints": {
+                "minimum": 0
+            }
+        },
+        {
+            "name": "end",
+            "type": "integer",
+            "description": "1-based position of the right-most position of sequence with respect to the reference chromosome",
+            "constraints": {
+                "minimum": 1
+            }
+        },
+        {
+            "name": "strand",
+            "type": "string",
+            "description": "strand of sequence in reference",
+            "constraints": {
+                "enum": [
+                    "+",
+                    "-"
+                ]
+            }
+        },
+        {
+            "name": "variant_class",
+            "type": "array",
+            "description": "the class of the variant(s), allowing for multiple variants to be tested in one sequence (haplotypes)",
+            "constraints": {
+                "enum": [
+                    "SNP",
+                    "indel"
+                ]
+            }
+        },
+        {
+            "name": "variant_pos",
+            "type": "array",
+            "description": "0-based position of the start of the normalized representation of the variant(s). integer within [0, len(sequence) - 1]",
+            "constraints": {
+                "minimum": 0
+            }
+        },
+        {
+            "name": "SPDI",
+            "type": "array",
+            "description": "0-based, validated SPDI representation of the variant(s), e.g. NC_000001.11:25253603:G:A"
+        },
+        {
+            "name": "allele",
+            "type": "array",
+            "description": "the allele of the variant(s) with respect to the referenece chromosome sequence",
+            "constraints": {
+                "enum": [
+                    "ref",
+                    "alt"
+                ]
+            }
+        },
+        {
+            "name": "info",
+            "type": "string",
+            "description": "any additional comment or information"
+        }
+    ],
+    "missingValues": [
```

Comment:
In the sample data file, the missing value is NA.

Reply:
fixed